proto/pocket/supplier/query.proto

```diff
@@ -7,34 +7,45 @@ import "google/api/annotations.proto";
 import "cosmos/base/query/v1beta1/pagination.proto";
 import "pocket/supplier/params.proto";
 import "pocket/shared/supplier.proto";
+import "pocket/supplier/claim.proto";
 
 option go_package = "github.com/pokt-network/poktroll/x/supplier/types";
 
 // Query defines the gRPC querier service.
 service Query {
-
+  
   // Parameters queries the parameters of the module.
   rpc Params (QueryParamsRequest) returns (QueryParamsResponse) {
     option (google.api.http).get = "/pocket/supplier/params";
-
+  
   }
-
+  
   // Queries a list of Supplier items.
   rpc Supplier    (QueryGetSupplierRequest) returns (QueryGetSupplierResponse) {
```

Comment:
Formatting, it seems like there's a lot of whitespace between `Supplier` and the open parenthesis?

Reply:
This formatting is done by ignite for some reason... Sadly gofmt doesn't do its magic in protobufs (at least AFAIK)